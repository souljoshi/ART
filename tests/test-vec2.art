void printfln(double x)
{
    printf(x);printc('\n');
}
int main()
{
    vec v1;
    vec v2;
    vec v3;

    v2 = <66.5, 12.0>;
    v3 = <1.0, 55.1>;

    v1[0] = 10.5;
    v1[1] = 4.2;

    v3[0] = 9.9;

    printfln(v1[0]);
    printfln(v1[1]);

    printfln(v2[0]);
    printfln(v2[1]);

    printfln(v3[0]);
    printfln(v3[1]);
}
