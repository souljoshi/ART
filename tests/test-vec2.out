10.500000
4.200000
66.500000
12.000000
9.900000
55.100000
